### PR TITLE
[Remove Stream/Optional usage] LPS-175651 Remove_Stream_usages_for_module_sharepoint_soap_repository

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Dante,

Resend: https://github.com/liferay-core-infra/liferay-portal/pull/3514
This is for ticket https://issues.liferay.com/browse/LPS-175651.
Thanks for checking!

### CODE REVIEW

File: modules/dxp/apps/sharepoint-soap/sharepoint-soap-repository/src/main/java/com/liferay/sharepoint/soap/repository/connector/internal/util/test/SharepointConnectionTestUtil.java [hunk 1]
```diff
@@ -80,9 +79,20 @@ private static String _getSharepointVMHostname() throws Exception {
 		try (BufferedReader bufferedReader = new BufferedReader(
 				new InputStreamReader(httpURLConnection.getInputStream()))) {
 
-			Stream<String> payloadStream = bufferedReader.lines();
+			StringBundler sb = new StringBundler();
```
Comment (jeffwu0724) [inline modules/dxp/apps/sharepoint-soap/sharepoint-soap-repository/src/main/java/com/liferay/sharepoint/soap/repository/connector/internal/util/test/SharepointConnectionTestUtil.java:82]: changed to use StringBundler

**Comment by jeffwu0724** [comment]
ci:test:sharepoint

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by dantewang** [comment]
@jeffwu0724 See https://github.com/brianchandotcom/liferay-portal/pull/131706, just modified the message of the last commit.

File: modules/dxp/apps/sharepoint-soap/sharepoint-soap-repository/src/main/java/com/liferay/sharepoint/soap/repository/connector/internal/util/test/SharepointConnectionTestUtil.java [hunk 2]
```diff
@@ -80,9 +79,20 @@ private static String _getSharepointVMHostname() throws Exception {
...
 
-			String payload = payloadStream.collect(Collectors.joining());
+			String line = null;
+
+			while ((line = bufferedReader.readLine()) != null) {
+				sb.append(line);
+				sb.append(StringPool.BLANK);
```
Comment (dantewang) [inline modules/dxp/apps/sharepoint-soap/sharepoint-soap-repository/src/main/java/com/liferay/sharepoint/soap/repository/connector/internal/util/test/SharepointConnectionTestUtil.java:88]: @jeffwu0724 `Stream<String>.collect(Collectors.joining())` just concats all the strings, so here the `StringPool.BLANK` is not necessary.

You can verify it with a simple piece of code:

```
		List<String> strings = Arrays.asList("A", "B", "C");

		System.out.println(strings.stream().collect(Collectors.joining()));
```

outputs `ABC` not `A B C`.

**Comment by dantewang** [comment]
@jeffwu0724 Closed the pull sent to Brian. Please see my comment and resend, thanks!

**Pull request closed without merging**